services/storage/src/simcore_service_storage/modules/celery/worker_main.py

```diff
@@ -0,0 +1,88 @@
+"""Main application to be deployed in for example uvicorn."""
+
+import asyncio
+import logging
+import threading
+
+from asgi_lifespan import LifespanManager
+from celery import Celery
+from celery.signals import worker_init, worker_shutdown
+from servicelib.background_task import cancel_wait_task
+from servicelib.logging_utils import config_all_loggers
+from simcore_service_storage.modules.celery.utils import set_celery_worker
+
+from ...core.application import create_app
+from ...core.settings import ApplicationSettings
+from .common import create_app as create_celery_app
+from .example_tasks import sync_archive
+from .worker import CeleryTaskQueueWorker
+
+_settings = ApplicationSettings.create_from_envs()
+
+# SEE https://github.com/ITISFoundation/osparc-simcore/issues/3148
```

Comment:
I dont think this comment applies here, what exactly from this issue are you trying to reference here, is the logging you use *insecure*? Maybe clarify with respect to what one should `SEE` this issue when refactoring this file